kafka-client/src/main/java/dev/responsive/db/CassandraFactSchema.java

```diff
@@ -147,10 +152,12 @@ public MetadataRow metadata(final String table, final int partition) {
         .setInt(PARTITION_KEY.bind(), partition);
     final List<Row> result = client.execute(bound).all();
 
-    if (result.size() != 1) {
+    if (result.size() > 1) {
       throw new IllegalStateException(String.format(
-          "Expected exactly one offset row for %s[%s] but got %d",
+          "Expected at most one offset row for %s[%s] but got %d",
```

Comment:
we don't initialize the offsets in the case of global stores, so it's possible to get no rows back

Reply:
Will probably end up doing the same for actual WindowStores soon, so works for me